Vagrantfile

```diff
@@ -0,0 +1,65 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
```

Comment:
minor: could add `# frozen_string_literal: true` to be compatible with future Ruby versions.

Reply:
This is not really a full-fledged Ruby script, but sure, will do. Also, I'm not a Ruby dev, this file is mostly knit together from stuff found in the Vagrant docs :innocent: